docs/Controller Guide.md

```diff
@@ -0,0 +1,100 @@
+# NMOS Controller Implementation Guide
+
+## Scope
+This document is intended as a guide for implementers or users of Controllers within NMOS-enabled networked media systems. The document defines what a Controller is and outlines the requirements of a Controller with respect to each existing NMOS specification through references to the relevant sections of those documents.
+
+The document focuses primarily on guidance for the core NMOS specifications of IS-04, IS-05 and BCP-003-01. However, this is a living document and it is intended that fuller guidance for other NMOS specifications be added in future.
+
+## Use of Normative Language
+This document is a guide to Controllers only and not a specification itself. It provides informational guidance around the normative requirements of the relevant specifications. If there are any inconsistencies between this guide and the specifications then, unequivocally, the specification is correct. Any normative language key words found in this document are not to be interpreted as RFC 2119 key words.
+
+## Controller Definition
+A Controller is Client software that interacts with the NMOS APIs to discover, connect and manage devices within a networked media system. The diagram below shows some of those API interactions with other NMOS system components. 
+
+![Role of NMOS Controller in a Networked Media System](./images/NMOS_Controller.svg)
+
+The EBU have also produced a useful diagram showing the role of Controllers within a wider context (link to be added).
```

Comment:
When will the link be available?

Reply:
Not sure, so I have removed this line for now until the EBU diagram is publicly available.